exporter/splunkhecexporter/client_test.go

```diff
@@ -921,12 +921,6 @@ func TestInvalidJson(t *testing.T) {
 	assert.Error(t, err)
 }
 
-func TestStartAlwaysReturnsNil(t *testing.T) {
```

Comment:
Why this test is deleted?

Reply:
With this change, start is now meaningful, as it initializes the http client. So this test is no longer applicable.